README.md

```diff
@@ -5,6 +5,105 @@ on top of [Celery](https://docs.celeryproject.org/).
 * Celery tasks interface
 * Anti-flood tactics
 
+Installation
+------------
+
+```shell script
+pip install celery-amqp-events
+```
+
+Configuration
+-------------
+
+1. Set a "service name" for each service that has event handlers.
```

Comment:
how?

Reply:
Good question. Below is an example with `events_app` function.